src/Reducer.test.js

```diff
@@ -0,0 +1,299 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import { combinedReducer } from "./store/reducer";
+import {
+  UPDATE_SELECTED_STOCK,
+  UPDATE_SEARCH,
+  UPDATE_SEARCH_QUERY,
+  UPDATE_COMPANY_SYMBOLS,
+  UPDATE_CHART_RANGE,
+  UPDATE_KEY_STATS,
+  UPDATE_LATEST_NEWS,
+  UPDATE_COMPANY_OVERVIEW,
+  UPDATE_TOP_PEERS,
+  UPDATE_CHART_DATA,
+  UPDATE_STOCK_TICKER
+} from "./store/constants";
+
+// import { store } from "./store/store";
+// import { Provider } from "react-redux";
+
+// it("renders without crashing", () => {
+//   const div = document.createElement("div");
+//   ReactDOM.render(<Provider store={store}><App /></Provider>, div);
+//   ReactDOM.unmountComponentAtNode(div);
+// });
+
+describe("Stock reducer", () => {
+  let previousState;
```

Comment:
I'd definitely look at moving these `previousState` and `newState` variables to within each `describe` closure, just to make sure they're definitely not interacting cross-test.

Reply:
How would I place the `previousState` within each describe block? My previous state is an object containing the same elements as my `reducer`, so I thought that would need to be accessed by all describe blocks